### ISSUE TITLE
Using the Learn platform with the Documentation handbooks.

### ISSUE THREAD

**Issue by paaljoachim** [WordPress/Learn#159]
Hi

I am looking into improving the general Docs structure of the handbooks: 
https://make.wordpress.org/docs/handbook/

Which means I have decided to setup a dev prototype site where I plan on adjusting the structure. I have been spending much of the day looking at various doc/wiki plugins and notice various advantages and disadvantages of each.

What we really should do is use the Learn platform (plugin and theme) for Documentation and for Learning. It would make it a lot easier for the training and docs team to work together. We would have one Documentation and Learning platform for WordPress with easier cross linking of docs <-> training.
It would also be easier to maintain. Being able to improve one plugin and one theme that works for both teams. 

I would like to in a simple way download the plugin and theme and test it out. At the moment from reading the readme it feels a bit complex getting it to work on a dev site, as it requires more than just downloading and installing the theme and plugin. Thank you!

**Comment by coreymckrill** [comment]
@paaljoachim The plugin (and less so the theme, I think) are dependent on some other plugins, some of which are 3rd party and some of which are custom for the WordPress.org codebase, plus some custom database tables. This is why we're using `wp-env` for a shared local dev environment and composer/wp-cli for provisioning it, to make sure it has all the dependencies it needs. You can try copying the plugin/theme from this repo into your own WP install, but I'm not sure how well they'd work without all the other stuff.

**Comment by paaljoachim** [comment]
Thanks for the feedback, @coreymckrill  Corey!

Here is a recap of what I am working on.

1. Finding the handbooks in the current support page structure on wordpress.org is very difficult. The support page is a maze and the structure needs to be recreated.
I built this dev site with focus on structure:
https://wpdocs.easywebdesigntutorials.com/documentation-handbooks/
Showing all the handbook links on one page. (The design on the page is the default used by the WeDoc plugin I am using.)

2. I want to begin with basics of just making the handbooks font bigger. I then located this meta trac ticket.
https://meta.trac.wordpress.org/ticket/5353
Bottom line here is that the stylesheet used in the theme for wordpress.org is a mess.

3. Learn is experimenting with a new design/layout/structure. I believe a lot of this can be brought over in one form or another to other handbooks. Such as elements in the landing page and lesson pages.

**Comment by coreymckrill** [comment]
It doesn't seem like there is anything actionable here for the Learn site itself, so I'm going to close this. I would recommend opening a ticket on Meta Trac for any discussions about the wp.org theme's stylesheet.
